src/main/java/com/vmware/vipclient/i18n/messages/service/PatternService.java

```diff
@@ -63,7 +63,7 @@ public JSONObject getPatterns(String language, String region) {
 
     private JSONObject getPatternsFromBundle(String locale) {
         JSONObject patterns = null;
-        if (LocaleUtility.isDefaultLocale(locale)) {
+        if (LocaleUtility.isDefaultLocale(locale) || locale.contains("en")) {
```

Comment:
are you sure locale.contains("en") means the locale is default? A locale string could easily contain 'en'.

Reply:
Rolled back this cha nge